controller/imageController.js

```
@@ -0,0 +1,83 @@
+      roundedCornerRadius = 0;
+    }
+
+    let sharpImage = sharp(req.file.buffer);
+    let contentType = `image/${outputFormat}`;
+
+    switch (outputFormat) {
+      case "jpeg":
+        sharpImage = sharpImage.jpeg({ quality: quality, mozjpeg: true });
+        break;
+      case "png":
+        const pngCompressionLevel = Math.max(0, 9 - Math.floor(quality / 11));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)